ipalib/plugable.py

```diff
@@ -494,6 +495,13 @@ def build_global_parser(self, parser=None, context=None):
         """
         Add global options to an optparse.OptionParser instance.
         """
+        def config_file_callback(option, opt, value, parser):
+            if not ipautil.file_exists(value):
+                raise optparse.OptionValueError(
+                    _("%s: file not found") % value)
```

Comment:
Shouldn't be used this rather parser.error() instead of raising an exception? https://docs.python.org/2/library/optparse.html#how-optparse-handles-errors